pages/SignIn.js

```diff
@@ -0,0 +1,39 @@
+import BasePage from '../pages/BasePage';
+
+class SignIn extends BasePage {
+
+  //WebElements
+  get username(){ return $('[name="username"]') }
+  
+  get password() {return $('[name="password"]')}
+
+  get submitbtn() {return $('#submit')};
+
+  get text() {return $('span')};
+
+  get checkbox() {return $('#remember-me')};
+
+  get signUpForm() {return $('a[href="/bank/signup"]')}
+
+  get logo() {return $('a[href="/bank/home"]')}
+
+  get headerSignIn() {return $('strong')}
```

Comment:
Similar to previous comment, this is an html tag that is not really correlated to the element you are locating, as **strong** is used mainly for giving **boldness* style to elements (something that can very well used in any other element, and is not an inherent attribute of this element in particular). As with the `span` example, this has high risk of breaking if developers decide to **bold** any other text in the page in the future. Craft another selector that is more closely related to this specific element, so you cover for risk of collisions in the future.